### ISSUE TITLE
Refresh button for mech listings

### ISSUE THREAD

**Issue by VictorMorson** [MegaMek/megamek#86]
I apologize if this is the incorrect place to post a feature request; I know this has been something excessively frustrating to myself and everyone who I avidly play with.  When you create or modify a BLK, the only option to update it in the mech listings is to reboot the game.  When you are making a lot of units (or trying funky things), this can get very frustrating - for example if I want to customize one unit in a company.

Long story short it would be HIGHLY useful to have a simple "Refresh" button that rescans the BLKs without having to completely close out and reload the game.

Again sorry if this is in the incorrect forum.  I just know this is something that over the years has probably caused considerable delays and general difficulty and it'd be great to see it updated.

**Comment by arlith** [comment]
Right now, we don't have a lot of policy information up for posting issues.  I've started trying to get some policy up, but it's an ongoing process.  In GitHub, instead of separate trackers, there's just one "issue" tracker.  You can add labels, and one of the labels is "enhancement".  Features should go under this label.  I believe users can add those labels themselves.

As for the feature itself, this has been requested before, and probably wouldn't be this tough.  I'd like to get to this if I get some time, but that's been in short supply.

**Comment by pheonixstorm** [comment]
Actually you (arlith) or one of the other devs have to assign a label to each issue. The rest of us can only create new issues or close our own but we can not assign labels to tickets we create.

**Comment by Dylan-M** [comment]
I've always found the requests for this feature confusing and funny, since everyone I know of does their modifications between games rather than when trying to start a game. Especially when using MHQ. However, as Arlith said, this should be relatively easy.

**Comment by mkerensky** [comment]
Using the current system, I have to delete my units.cache file when editing a BLK or other unit file, otherwise I end up with multiple versions of the same unit.  Something to watch out for with this feature.

**Comment by VictorMorson** [comment]
This is awesome if it does happen.  Largely we run into it when trading our forces custom units between other players, forcing them to reload the entire game and reset their units if the original player isn't there to trade the unit.  I also run into it when assembling forces and want to give a pilot some minor modifications; it's not game breaking but I've easily reset Megamek 100+ time setting up some long term scenarios.

It's a really cool feature if it can be made to work. Fingers crossed for luck on this one.
